app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -152,7 +152,11 @@ Fae.form.filtering = {
    */
   sortingSetup: function() {
     this._addSortingClasses();
-    this._addSortingListeners();
+
+    if (!this._are_sorting_listeners_setup) {
+      this._addSortingListeners();
+      this._are_sorting_listeners_setup = true;
```

Comment:
Good call. Looking back at it, I'll try moving the check for `this._are_sorting_listeners_setup` into `_addSortingListeners()` and have it trigger an early return if set.